dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_item.tsx

```diff
@@ -185,8 +186,8 @@ export const DataConfigPanelItem = ({ fieldOptionList, visualizations }: any) =>
     return sectionName === 'metrics'
       ? unselectedFields
       : visualizations.vis.name === visChartTypes.Line
-        ? unselectedFields.filter((i) => i.type === 'timestamp')
-        : unselectedFields;
+      ? unselectedFields.filter((i) => i.type === 'timestamp')
+      : unselectedFields;
```

Comment:
I think the indent was correct previously.

Reply:
I tried indenting it and used prettify then. It's getting back to this indentation.